src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -25,9 +24,9 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     internal class HttpApiHandler
     {
         // Route segments
-        private const string InstancesControllerSegment = "/instances/";
-        private const string OrchestratorsControllerSegment = "/orchestrators/";
-        private const string EntitiesControllerSegment = "/entities/";
+        private const string InstancesControllerSegment = "instances/";
+        private const string OrchestratorsControllerSegment = "orchestrators/";
+        private const string EntitiesControllerSegment = "entities/";

```

Comment:
Any particular reason for these changes?

Reply:
`TemplateMatcher` throws an exception when routes start with a `/` character in the version of `Microsoft.AspNetCore.Routing` we use in Functions V1.